gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,5 +1,5 @@
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-8.1.1-bin.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-8.6-bin.zip
```

Comment:
Why not gradle 8.7?
https://github.com/gradle/gradle/releases

Reply:
Will switch to it on next release. Honestly, idk why I have ignored 8.7

Anyway it affects nothing, except features like increment compiling and so on

Reply:
Also patches a High-severity CVE: https://github.com/asyncapi/jasyncapi-idea-plugin/issues/61